### PR TITLE
Update DeveloperGuide.md

### PR DESCRIPTION
**Pull request by rockman007372**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -381,28 +391,27 @@ Use case ends.
...
+  * *a1. User requests to end the review session.  
+  * *a2. Powercard ends the session and brings the user back to the area before the review started.  
+    Use case ends.
 
 ### Non-Functional Requirements
 
-1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
-2.  Should be able to hold up to 1000 Powercards without a noticeable sluggishness in performance for typical usage.
+1. Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
+2. Should be able to hold up to 1000 Powercards without a noticeable sluggishness in performance for typical usage.
```
Comment (ruiyigan) [inline docs/DeveloperGuide.md:409]: HAHAHA must we populate and test this?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -381,28 +391,27 @@ Use case ends.
...
 3. A Powercard should load when prompted without any noticeable lag
 4. Every command should provide a response within 2 seconds
 5. The Powercard program is not expected to determine the correctness of the user’s response
 6. A Powercard should be easily added/deleted in less than 3 commands after opening the program
```
Comment (ruiyigan) [inline docs/DeveloperGuide.md:413]: Ours is maximum 3 commands right. addDeck, selectDeck, add.

**Review by ruiyigan — APPROVED** [review]
LGTM!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (rockman007372) [reply docs/DeveloperGuide.md:409]: LMAO lets find out

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (rockman007372) [reply docs/DeveloperGuide.md:413]: yep!

**Pull request merged**